crates/kdc/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "kdc"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+picky-asn1 = { workspace = true, features = ["time_conversion"] }
+picky-asn1-der.workspace = true
+picky-asn1-x509.workspace = true
```

Comment:
I’m really sorry to say that now, but we’ll stop using the picky crates soon in the future in favor of using the `der` crate from RustCrypto. Still, I think it’s fine to merge this code now that it’s here, but we should stop writing any new code using this infrastructure from now on. If it’s not too much work for you, you may consider changing that before we merge.

Reply:
> If it’s not too much work for you, you may consider changing that before we merge.

To remove `picky-asn1-*` usage in the `kdc` implementation, we need to migrate the `picky-krb` crate to `der`-based encoding/decoding. It will significantly delay the credentials-injection feature.

I would like to address it in a separate PR. We need to refactor the `picky-krb` first.